pages/blog/index.vue

```diff
@@ -21,15 +25,15 @@
                 class="text-nuxt-green underline"
                 to="/team#nuxtCompany"
               >
-                NuxtJS team
+                {{ $t('blog.nuxt_team') }}
               </NuxtLink>
             </template>
             <template v-slot:ambassadors>
               <NuxtLink
                 class="text-nuxt-green underline"
                 to="/team#nuxtCommunity"
               >
-                Nuxt.js Community
+                {{ $t('blog.ambassadors') }}
```

Comment:
this should be community. community includes ambassadors but it is more than just ambassadors